internal/controller/kbsconfig_controller.go

```diff
@@ -330,20 +330,37 @@ func (r *KbsConfigReconciler) newKbsDeployment(ctx context.Context) *appsv1.Depl
 		kbsDeploymentType = confidentialcontainersorgv1alpha1.DeploymentTypeMicroservices
 	}
 
-	// RunAsUser (root) 0
-	runAsUser := int64(0)
```

Comment:
Can we use a specific userid (eg 1001) here ?
It'll also require changes to the Dockerfile to use a specific USER (ie 1001) as well as the deployment spec to set the securityContext. I think this will ensure it works across k8s distros without requiring any specific policies that allows any non-root userid in the container

Reply:
I would recommend creating a new method to create security context that is something like
```
runAsUser := int64(1000)
runAsGroup := int64(1000)

SecurityContext: &corev1.SecurityContext{
			RunAsUser:                &runAsUser,
                         RunAsGroup:             &runAsGroup,
			AllowPrivilegeEscalation: ptr.Bool(false),
			Capabilities: &corev1.Capabilities{
				Drop: []corev1.Capability{
					"ALL"},
			},
                        SeccompProfile: &corev1.SeccompProfile{
				Type: corev1.SeccompProfileTypeRuntimeDefault,
			},
                        
		}
```